marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/flow.sjs

```diff
@@ -349,9 +349,10 @@ class Flow {
     if (!hook || !hook.module) {
       hook = processor.customHook;
     }
+    let parameters;
```

Comment:
This is a change in how custom hooks work, and I've wondered before why it doesn't work this way - i.e. I think this change is correct. However, I think it should be handled via a separate ticket so we can think through what tests are needed for this.

Reply:
The ticket would have a scope of verifying that a "after" custom hook has access to the content array that's returned by the step module. Again, that's probably how it always should have worked. But we should treat the change as a bug as it is changing the behavior in a way that could be view as a backwards incompatibility, but in way that we think was intended all along.